source/css/_common/components/highlight/highlight.styl

```diff
@@ -110,14 +107,24 @@ pre {
   td { border: none; }
 }
 
-// For diff highlight
-pre .deletion { background: $highlight-deletion; }
-pre .addition { background: $highlight-addition; }
-pre .meta     { color: $highlight-purple; }
-
 pre {
+  // For diff highlight
+  .deletion {
+    background: $highlight-deletion;
+  }
+
+  .addition {
+    background: $highlight-addition;
+  }
 
-  .comment { color: $highlight-comment; }
+  .meta {
+    color: $highlight-yellow;
+    disable-user-select();
```

Comment:
Only for shell lang, bash doesn't have `meta` class

Reply:
Otherwise we need to remind users to use shell for $

Reply:
https://github.com/theme-next/hexo-theme-next/issues/448